tests/generate_packageTest.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
oh they are! weird. the last time i checked they were not. this is weird, maybe the link wasnt updated or something ... forget it :D thanks